cmake/findDependencies.cmake

```diff
@@ -35,6 +35,9 @@ else()
     endif ()
 endif()
 
+if( EXIV2_ENABLE_HEIF )
```

Comment:
It seems like you are not using the 3rd party library. In that case we would not need to find it and we would not need to add the definition `EXV_ENABLE_HEIF` to control whether we support that file format or not. We could always support it.

Reply:
I think you should offer a way to disable this, because patents.

Reply:
The option `EXIV2_ENABLE_HEIF` is disabled by default.